src/ble/CHIPBleServiceData.h

```diff
@@ -45,11 +45,14 @@ enum chipBLEServiceDataType
  */
 struct ChipBLEDeviceIdentificationInfo
 {
-    constexpr static uint16_t kDiscriminatorMask     = 0xfff;
-    constexpr static uint8_t kAdditionalDataFlagMask = 0x1;
+    constexpr static uint16_t kDiscriminatorMask       = 0xfff;
+    constexpr static uint8_t kAdditionalDataFlagMask   = 0x1;
+    constexpr static uint8_t kAdvertisementVersionMask = 0x0f;
+    constexpr static uint8_t ShiftBits                 = 4u;
 
     uint8_t OpCode;
-    uint8_t DeviceDiscriminator[2];
+    uint8_t
+        DeviceDiscriminatorAndAdvVersion[2]; // from lsb 0-11 bits for device discriminator and 12-15 bits for advertisement version
```

Comment:
This is an array of two bytes.  Which of these bits are where in these bytes?  That's what needs to get documented.